src/dialog/createplaylistdialog.cpp

```diff
@@ -0,0 +1,75 @@
+#include "createplaylistdialog.hpp"
+
+#include <QFormLayout>
+#include <QDialogButtonBox>
+#include <QPushButton>
+#include <QMessageBox>
+
+CreatePlaylistDialog::CreatePlaylistDialog(QWidget *parent)
```

Comment:
I would much prefer to try to merge this with the dialog for editing playlists. Or maybe even better, try to do it like the official client does, that is, only lets you choose a name for the playlist before creating. Other stuff like public, description, etc. have to be changed after creation.

Reply:
By that you mean, spawn a QInputDialog for entering the name and add a single track to the playlist?
That would then eliminate the need of the separate dialog and cohere with the behavior of the official client.